classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label = '')
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
+  end
```

Comment:
- [ ] When adding a student to a classroom it also sets the classroom for the student:

Something as follow:

```ruby
  def add_student(student)
    @students << student
    student.classroom = self
  end
```

Reply:
Done